src/repeated_game.jl

```diff
@@ -254,30 +254,54 @@ Given a constraint w ∈ W, this finds the worst possible payoff for agent i.
    here `nH` is the number of subgradients.
 - `C::Vector{Float64}` : The array containing the hyperplane levels.
 - `i::Int` : The player of interest.
-- `lp_solver::AbstractMathProgSolver` : Allows users to choose a particular
-  solver for linear programming problems. Options include ClpSolver(),
-  CbcSolver(), GLPKSolverLP() and GurobiSolver(). By default, it choooses
-  ClpSolver().
+- `lp_solver::Union{MathOptInterface.AbstractOptimizer,Function}` : Linear
```

Comment:
This should be

```suggestion
- `lp_solver::Union{Type{<:MathOptInterface.AbstractOptimizer},Function}` : Linear
```

(The same for the docstring for `is_dominated`.)